Signal/src/ViewControllers/CodeVerificationViewController.m

```diff
@@ -227,12 +231,18 @@ - (void)createViews {
     [_requestCallSpinner autoPinEdgeToSuperviewEdge:ALEdgeRight withInset:kSpinnerSpacing];
 }
 
-- (void)updatePhoneNumberLabel {
-    NSString *phoneNumber = [PhoneNumber bestEffortFormatPartialUserSpecifiedTextToLookLikeAPhoneNumber:[TSAccountManager localNumber]];
+- (NSString *)phoneNumberText
+{
     OWSAssert([TSAccountManager localNumber] != nil);
-    _phoneNumberLabel.text = [NSString stringWithFormat:NSLocalizedString(@"VERIFICATION_PHONE_NUMBER_FORMAT",
-                                                                         @"Label indicating the phone number currently being verified."),
-                             phoneNumber];
+    return [PhoneNumber bestEffortFormatPartialUserSpecifiedTextToLookLikeAPhoneNumber:[TSAccountManager localNumber]];
```

Comment:
Oh, I see what you're saying. If I specified the users country code, we can be sure we aren't just formatting their number in their default locale.